index.html

```diff
@@ -199,33 +199,36 @@ <h2>Skills</h2>
             <header class="title">I'm always interested in hearing about new projects,<br>
                 so if you'd like to chat please get in touch.
             </header>
-            <div class="center">
-                <form action="#">
-                    <input class="input" type="text" id="fname" placeholder="   Full Name"><br><br>
-                    <input class="input" type="text" id="lname" placeholder="   Email Address"><br><br>
-                    <textarea class="font" id="textareaid" name="textareaname" rows="4" cols="50" maxlength="200">  Enter text here.
-                    </textarea><br><br>
-                    <input class="see-project font" type="submit" value="Get in touch">
-                </form>
-            </div>
-            <div class="center margin-top-25px">
-                <a href="">
-                    <img class="margin-10px" src="./assets/twitter.svg" alt="twitter">
-                </a>
-                <a href="">
-                    <img class="margin-10px" src="./assets/linkedin.svg" alt="linkedin">
-                </a>
-                <a href="">
-                    <img class="margin-10px" src="./assets/medium.svg" alt="medium">
-                </a>
-                <a href="">
-                    <img class="margin-10px" src="./assets/github.svg" alt="github">
-                </a>
-                <a href="">
-                    <img class="margin-10px" src="./assets/angellist.svg" alt="angellist">
-                </a>
-            </div>
-            <p class="font center mailme">htetnaing0814@gmail.com</p>
+            <footer>
+                <div class="center">
+                    <form action="https://formspree.io/f/moqrrpoj" method="POST">
```

Comment:
- [ ] When the form submits the network gets only the message. But not the name and email. Kindly fix it